unified_planning/model/problem.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, name: str = None, env: 'up.environment.Environment' = None, *
         self._timed_goals: Dict['up.model.timing.TimeInterval', List['up.model.fnode.FNode']] = {}
         self._goals: List['up.model.fnode.FNode'] = list()
         self._metrics: List['up.model.metrics.PlanQualityMetric'] = []
+        # The field initial default optionally associates a non-user type to a default value. When a new fluent is
```

Comment:
Why only `non-user` type?

Reply:
@arbimo I updated your comment removing the `non-user`, since it is possible to define the default also for a `UserType`.

Reply:
That's right, thank you.